packages/css-library/src/utilities/flex-grid.scss

```diff
@@ -51,4 +51,4 @@ $u-flex-grid-cols: (
   ),
 );
 
-$u-flex-grid: map-collect($u-flex-grid-cols, $u-flex-grid-cols);
+$u-flex-grid: map-collect($u-flex-grid-col, $u-flex-grid-cols);
```

Comment:
Just a small bug that didn't get caught in the initial commit. We should see now that the `vads-l-col` utilities are generating.